fs_brokers/apache_hdfs_broker/src/main/java/org/apache/doris/broker/hdfs/FileSystemManager.java

```
@@ -511,7 +598,7 @@ public void pwrite(TBrokerFD fd, long offset, byte[] data) {
             long currentStreamOffset;
             try {
                 currentStreamOffset = fsDataOutputStream.getPos();
-            } catch (IOException e) {
+            } catch (Exception e) {
```

Why catch all exception

---

should be 'IOException'